packages/ckeditor5-core/src/command.ts

```diff
@@ -121,9 +121,15 @@ export default class Command extends ObservableMixin() {
 				return;
 			}
 
+			const selection = editor.model.document.selection;
+			const selectionInGraveyard = selection.getFirstPosition()!.root.rootName == '$graveyard';
+			const canEditAtSelection = !selectionInGraveyard && editor.model.canEditAt( selection );
```

Comment:
I believe this was my initial approach as it totally makes sense. Unfortunately, I don't remember at this moment why it landed here after all (and this was changed quite some time ago). I wish I made a comment about that.

If it is not there, then maybe `canEditAt()` is used at some place where it should allow to make changes in graveyard.

Now as I think of it, it might be undo. In undo we check if we can perform an operation based on `canEditAt()`. If it returns `false` we skip the operation. Most probably, if it returns `false` for graveyard, it would mess up with undo (you would not be able to undo insertions).